mcp-server/polaris_mcp/server.py

```diff
@@ -59,13 +59,30 @@
     "required": ["isError"],
     "additionalProperties": True,
 }
+DEFAULT_HTTP_RETRIES_TOTAL = 3
+DEFAULT_HTTP_RETRIES_BACKOFF_FACTOR = 0.5
+HTTP_RETRIES_STATUS_FORCELIST = [401, 409, 429, 500, 502, 503, 504]
```

Comment:
I think the write API retry should be more cautious. @singhpk234 has a PR in Iceberg to not retry on 50x.

Reply:
Not a blocker: we could retry for read endpoint with 5xx. We don't have to change that now. And we might not need to handle retry once we integrate with pyIceberg.